http.c

```diff
@@ -1078,8 +1079,8 @@ Datum http_request(PG_FUNCTION_ARGS)
 		elog(ERROR, "http_request.method is NULL");
 	method_str = TextDatumGetCString(values[REQ_METHOD]);
```

Comment:
Sorry seeing this just in the patch w/o context, want to confirm we have a null check?